ext/omnibus/D0370R3.html

```diff
@@ -49,12 +49,12 @@
   <tr>
     <td width="172" align="left" valign="top">Document number:</td>
     <td width="435">
-      D0370R2
+      D0370R3
```

Comment:
So that the document in the post-meeting mailing has *exactly* the wording that was incorporated into the working draft, including editorial changes.